draft-ietf-mls-protocol.md

```diff
@@ -2729,6 +2729,12 @@ itself) and S the other child:
 * D is in the resolution of C, and the intersection of P's `unmerged_leaves`
   with the subtree under C is equal to the resolution of C with D removed.
 
+* If D is a parent node, the `unmerged_leaves` field of D is a subset of the
+  `unmerged_leaves` field of P.
```

Comment:
This is implied by the property just above it
See the [discussion about the property](https://github.com/mlswg/mls-protocol/pull/713#issuecomment-1146399188) in #713, in which I show that the parent-hash link between D and P implies that `intersect(P.unmerged_leaves, subtree(D)) = D.unmerged_leaves`, which implies your condition.